### PR TITLE
api & ui. some fixes and new support to ikev2

### PR DESCRIPTION
**Pull request by edospadoni**
- added validation on IPSEC create/update on left an right subnets fields
- added `red` alias as an option to select on local IP
- added `ikev2` option selectable in advanced configuration

https://github.com/NethServer/dev/issues/6049

### CODE REVIEW

**Review by gsanchietti — APPROVED** [review]

**Comment by gsanchietti** [comment]
Waiting for the issue from @cotosso before merge

**Comment by mamengoni** [comment]
Tested and verified:

Schermata da 2020-02-05 09-55-35
Schermata da 2020-02-05 09-55-18
Schermata da 2020-02-05 09-55-05

**Pull request merged**
